bin/vyper

```diff
@@ -20,7 +17,7 @@ if tb_limit:
 
 parser = argparse.ArgumentParser(description='Vyper programming language for Ethereum')
 parser.add_argument('--version', action='version', version='{0}'.format(vyper.__version__))
-parser.add_argument('input_file', help='Vyper sourcecode to compile')
+parser.add_argument('input_files', help='Vyper sourcecode to compile', nargs='+')
 parser.add_argument('-f', help='Format to print', default='bytecode', dest='format')
```

Comment:
can you add a list of options here so the help displays what possible choices are?

Reply:
@fubuloubu  I think we are ready to merge now? :man_dancing: